exercises/01-react-stateful/src/modules/users/components/users-list.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { User } from '../users-types'
+
+export const UsersList: React.FC = () => {
+  // Declare a new state variable called 'users'
+  const [users, setUsers] = useState<Array<User>>([])
+
+  // Adds a new user to the 'users' state
+  const addUser = (user: { firstName: string; lastName: string }) => {
+    setUsers([...users, { id: users.length + 1, ...user }])
```

Comment:
Better to use a callback with the previous state.

```suggestion
    setUsers(prevUsers => [...prevUsers, { id: prevUsers.length + 1, ...user }])
```

Reply:
Fixed. Thanks! Somehow I didn't realize that my previous fix wasn't merged. See 2a178657d7dc311b54e1b4337f4377ef63bf5abf